src/utils/useId.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
```

Comment:
copyright pls

Reply:
Ah geez. You can tell I don't get to add new files very often. 🤦‍♂️

Thanks, should be fixed now.